dart-common/src/main/java/dart/common/util/NavigationModelBindingTargetUtil.java

```diff
@@ -82,28 +82,33 @@ private void checkForParentsOutside(
       final NavigationModelBindingTarget navigationModelBindingTarget = target.getValue();
       // root inside module
       if (navigationModelBindingTarget.parentPackage == null) {
```

Comment:
you should add a small comment here to say we use this field as caching.

Reply:
added a comment explaining a little bit more how it works